languages/rust/src/lf.rs

```diff
@@ -207,6 +213,12 @@ impl From<LfReturn> for u64 {
     fn from(ret: LfReturn) -> Self { ret.0 as u64 }
 }
 
+pub fn current_device() -> _lf_device {
```

Comment:
I don't think we should have a public function which returns a raw pointer. This should either be `pub(crate)` or we should find a way to do without this entirely.

Reply:
flipper::attach() returns a raw pointer though